src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1327,7 +1327,7 @@ def download_result(self, path=None, progress_bar=False, preference=None):
 
         preference : str
           Specify the preferred result file, either ``rst`` or ``rth``.
-          Only required when both files are present. Defaults to 'rst'.
+          Only required when both files are present. The default is 'rst'.
```

Comment:
```suggestion
          This parameter is only required when both files are present. The default is ```None``,
          in which case ``"rst"`` is used.
```
The first line shouldn't start with a verb. It should be something like "Preferred type for the result file, which is either ``"rst"`` or ``"rth"``.  Strings should use the double back tag for the literal string and then quotation marks.